lib/csrf.ts

```diff
@@ -7,12 +7,10 @@ class Csrf implements Authorizer {
   constructor(csrfHeader?: String, getToken?: Function) {
     this.getToken = getToken || Csrf.getTokenDefault;
     this.csrfHeader = csrfHeader || 'X-CSRF-TOKEN';
-    this.authorize = this.authorize.bind(this);
   }
 
-  public authorize(request: Function): Promise<any> {
-    return request([this.csrfHeader, this.getToken()]);
-  }
+  public authorize = (request: Function): Promise<any> =>
```

Comment:
Details about this in arrow functions.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#cannot_be_used_as_methods

Reply:
This matches the style in oauth.ts